portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerItTest.scala

```diff
@@ -52,5 +52,36 @@ class ProjectControllerItTest
           )
       }
     }
+
+    "updateProject" should {
+      "return status code NoContend if project was successfully updated" in {
+        val dummyProject = TestProjectUtils.getDummyProject(ProjectId(UUID.randomUUID().toString),UserId(ownerId))
```

Comment:
Why do you need to specify `ProjectId` here? `getDummyProject` can take care of parameters that you don't care about.

Reply:
I need order of parameters in this constructor. I need to explicitly specify userId. Without projectId I'll have type mismatch.

Reply:
You can use named arguments to specify only `userId`.
`TestProjectUtils.getDummyProject(userId = UserId(ownerId))`

Reply:
I don't know why, but If I do not specify projectId directly, then my tests fall because such a project already exists

Reply:
Because you did add a project with the same `projectId` in the previous test.
Unfortunately, `getDummyProject` returns projects with the same `projectId` every time you call it.
You should either fix method `getDummyProject` to make it return project with new `projectId` every time, or fix your tests to ensure that they will never affect each other. Fixing `getDummyProject` project is much easier, but you are free to choose the second option if you want :)

Reply:
fixed